examples/parallelReduce-api-counter/index.mjs

```diff
@@ -0,0 +1,41 @@
+import {loadStdlib} from '@reach-sh/stdlib';
+import * as backend from './build/index.main.mjs';
+const stdlib = loadStdlib(process.env);
+
+const startingBalance = stdlib.parseCurrency(100);
+
+const accs = await stdlib.newTestAccounts(4, startingBalance);
+const [accA, acc1, acc2, acc3] = accs;
+const ctcA = accA.contract(backend);
+
+const startUp = async () => {
+  const flag = "startup incomplete";
+  try{
```

Comment:
```suggestion
  try {
```

Reply:
Also, this code is `withDisconnect`, which you may want to teach about: https://docs.reach.sh/frontend/#p_148

Reply:
I'll add this with an iteration